plugins/com.google.cloud.tools.eclipse.appengine.validation/src/com/google/cloud/tools/eclipse/appengine/validation/AppEngineWebMarkerResolutionGenerator.java

```diff
@@ -29,7 +29,7 @@
  * Returns possible resolutions for given resource marker. Quick Fix resolutions
  * are available in the Problems view.
  */
-public class AppEngineWebMarkerResolutionGenerator implements IMarkerResolutionGenerator {
+class AppEngineWebMarkerResolutionGenerator implements IMarkerResolutionGenerator {
```

Comment:
This implements `org.eclipse.ui.IMarkerResolutionGenerator` and is referred to in `plugin.xml`. This might be read at runtime by external components? Is it safe to make it non-static?

Reply:
The javadoc on [`IConfigurationElement#createExecutableExtension()`](https://git.eclipse.org/c/equinox/rt.equinox.bundles.git/tree/bundles/org.eclipse.equinox.registry/src/org/eclipse/core/runtime/IConfigurationElement.java?h=master#n52) says:
> The named attribute value must contain a fully qualified name
> of a Java class. The class can either refer to a class implementing
> the executable extension or to a factory capable of returning the
> executable extension.
> 
> The specified class is instantiated using its 0-argument public constructor.

Reply:
It's still a tad ambiguous. For safety's sake, I'm going to restore public on these two classes for now.